securitycenter/snippets_v2/snippets_mute_config_v2.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START securitycenter_create_mute_config_v2]
+
+
+def create_mute_rule(parent_path: str, location_id: str, mute_config_id: str):
+    """
+     Creates a mute configuration under a given scope that will mute
+     all new findings that match a given filter.
+     Existing findings will NOT BE muted.
+     Args:
+         parent_path: use any one of the following options:
+                      - organizations/{organization_id}
+                      - folders/{folder_id}
+                      - projects/{project_id}
+         mute_config_id: Set a unique id; max of 63 chars.
+     """
```

Comment:
Please add the docstring for the returned value

Reply:
addressed